ee/vellum_ee/workflows/display/nodes/vellum/tests/test_final_output_node.py

```diff
@@ -78,3 +80,49 @@ class Outputs(BaseWorkflow.Outputs):
 
     # AND the serialized workflow should still be created
     assert "workflow_raw_data" in serialized_workflow
+
+
+def test_final_output_node_display__serialize_with_list_str_type():
+    """
+    Tests that FinalOutputNode with list[str] as second type parameter serializes correctly.
+    """
+
+    # GIVEN a node that outputs a list of strings
+    class ListNode(BaseNode):
+
+        class Outputs:
+            result: List[str]
```

Comment:
DevinAI update this to lowercase `list[str]` not the one from `typing`. If things break as a result, let me see the break